### DISCUSSION TITLE
LSPosed abadoned

### DISCUSSION THREAD

**Discussion by tomaszk8266** [Mahmud0808/Iconify#462] [General]
LSPosed is abadoned since three months. I'm using [this fork](https://github.com/mywalkb/LSPosed_mod), it is still well maintained and seems to be the future of xposed. I think that deprecation of original LSPosed should be indicated in readme, maybe with a recommendation to use this fork instead. What's your opinion about it @Mahmud0808?

**Comment by Mahmud0808** [comment]
You are right. You can make a PR for it if you want.

Edit: done
